Makefile

```
@@ -395,6 +396,48 @@ gloo-docker: $(GLOO_OUTPUT_DIR)/gloo-linux-$(GOARCH) $(GLOO_OUTPUT_DIR)/Dockerfi
 		-t $(IMAGE_REPO)/gloo:$(VERSION) $(QUAY_EXPIRATION_LABEL)
 
 #----------------------------------------------------------------------------------
+# Gloo with race detection enabled.
+# This is intended to be used to aid in local debugging by swapping out this image in a running gloo instance
+#----------------------------------------------------------------------------------
+GLOO_RACE_OUT_DIR=$(OUTPUT_DIR)/gloo-race
+
+$(GLOO_RACE_OUT_DIR):
+	mkdir -p $@
+$(GLOO_RACE_OUT_DIR)/Dockerfile.build: $(GLOO_DIR)/Dockerfile
+	cp $< $@
```

```suggestion
$(GLOO_RACE_OUT_DIR)/Dockerfile.build: $(GLOO_DIR)/Dockerfile
        mkdir -p $(GLOO_RACE_OUT_DIR)
	cp $< $@
```
i think the mkdir can just be added to this target since all the other targets call this one